seshdash/views.py

```diff
@@ -782,3 +781,43 @@ def graphs(request):
         return HttpResponseBadRequest()
 
 
+#function to create/edit sites
+@login_required
+def edit_settings(request):
+    site_available = []
+    #fetching list of all site
+    sites = Sesh_Site.objects.all()
+    for site in sites:
+        site_available.append(site.site_name)
+
+    form = SiteForm()
+
+    if request.method == 'POST':
+
+        form = SiteForm(request.POST)
+        #checking if the form is valid
+        if form.is_valid:
+
+            form = form.save()
+        else:
+            return HttpResponse("Didn`t validate")
```

Comment:
For the response you should return the form again. The form will automatically modified to show what the errors are in it.